ft_volumenormalise.m

```diff
@@ -150,6 +151,13 @@
   ft_error('the template anatomy should be stored in an SPM-compatible file');
 end
 
+if ~isempty(cfg.templatemask)
```

Comment:
please do this exactly the same as in the previous code block

Reply:
I am not sure... It is supposed to do the same as above?